lib/game_client/ogs/ogs_websocket_manager.dart

```diff
@@ -148,15 +148,13 @@ class OGSWebSocketManager {
         // Emit the message to listeners
         _messageController.add({
           'event': command,
-          'data': payload ?? {},
+          'data': payload,
           'request_id': requestId,
         });
-        
-        _handleSpecialEvents(command, payload ?? {});
```

Comment:
I just deleted this method altogether.  It was just logging, too coupled to games, and the "authentication" messages may have been hallucinated.